### PR TITLE
Improvements/chrome extension

### PR DESCRIPTION
**Pull request by JoFrost**

### CODE REVIEW

File: lavague-core/lavague/core/base_driver.py [hunk 1]
```diff
@@ -108,6 +108,22 @@ def get_html(self, clean: bool = True) -> str:
...
+        0 - Overview - OpenAI API
+        1 - [CURRENT] Nos destinations Train - SNCF Connect
+        """
+        pass
+
+    @abstractmethod
+    def switch_tab(self, tab_id: int) -> None:
+        """Switch to the tab with the given id"""
+        pass
+
```
Comment (adeprez) [inline lavague-core/lavague/core/base_driver.py]:
```suggestion
```
get_tabs and switch_tabs are already defined as non-abstract in the same class

Comment (JoFrost) [reply lavague-core/lavague/core/base_driver.py]: It was a merge issue, it's now gone

File: .gitignore [hunk 2]
```diff
@@ -67,6 +67,3 @@ test.ipynb
 #virtual environemnt
 env/
 venv/
-
-tests/*.py
-tests/*.ipynb
```
Comment (adeprez) [inline .gitignore]: We can keep these ignores

Comment (JoFrost) [reply .gitignore]: Restored as well

File: extension_chrome/src/background.ts [hunk 3]
```diff
@@ -1,8 +1,61 @@
...
+        const tabId = sender.tab?.id;
+        if (!tabId) {
+            console.error("No tabId found in the sender");
+            sendResponse({ response: {} });
+        }
+        else {
+            const domAction = new DomActions(tabId!);
+            (async () => {
+                try {
+                let eventsXpath: { [key: string]: string[] } = {};
```
Comment (adeprez) [inline extension_chrome/src/background.ts:20]: Minor style note: lines 21 to 47 need an extra level of indentation

**Pull request merged**